core/modules/widgets/button.js

```diff
@@ -91,7 +91,14 @@ ButtonWidget.prototype.render = function(parent,nextSibling) {
 			handled = true;
 		}
 		if(self.actions) {
-			self.invokeActionString(self.actions,self,event);
+			var modifierKey = event.ctrlKey && !event.shiftKey && !event.altKey ? "ctrl" : 
+				event.shiftKey && !event.ctrlKey && !event.altKey? "shift" : 
+                		event.ctrlKey && event.shiftKey && !event.altKey ? "ctrl-shift" : 
+				event.altKey && !event.shiftKey && !event.ctrlKey ? "alt" : 
+				event.altKey && event.shiftKey && !event.ctrlKey ? "alt-shift" : 
+				event.altKey && event.ctrlKey && !event.shiftKey ? "ctrl-alt" : 
+				event.altKey && event.shiftKey && event.ctrlKey ? "ctrl-alt-shift" : "normal";
+			self.invokeActionString(self.actions,self,event,{modifier: modifierKey});
```

Comment:
Did you test them with all OSes. Windows, unix, MacOS?

Reply:
I recommend either modelling after how this is done in the droppable widget, or also updating that widget to match for the sake of consistency: 
https://github.com/Jermolene/TiddlyWiki5/blob/master/core/modules/widgets/droppable.js#L133

Reply:
tested with linux and windows, not macOS @pmario

Reply:
@saqimtiaz you're right. I wonder what's been the reasoning for omitting the alt-key in droppable.js ... maybe it was a macOS thing